x-pack/plugins/ingest_manager/dev_docs/indexing_strategy.md

```diff
@@ -6,48 +6,48 @@ Overall documentation of Ingest Management is now maintained in the `elastic/sta
 Ingest Management enforces an indexing strategy to allow the system to automatically detect indices and run queries on it. In short the indexing strategy looks as following:
 
 ```
-{dataset.type}-{dataset.name}-{dataset.namespace}
+{data_stream.type}-{data_stream.dataset}-{data_stream.namespace}
```

Comment:
@ruflin could you look over the changes to this doc to make sure it's still accurate?